web-app/src/components/BookingModal/BookingModalForm/container.js

```diff
@@ -134,8 +136,8 @@ export default Wrapped =>
           }
           updateBookingAndDuration={this.props.updateBookingAndDuration}
           submitHolidayRequest={e => this.handleMakeHolidayRequest(e)}
-          updateHolidayRequest={e => this.handleUpdateHolidayRequest(e, true)}
-          deleteHolidayRequest={e => this.handleUpdateHolidayRequest(e, false)}
+          updateHolidayRequest={e => this.handleUpdateHolidayRequest(e, false)}
+          deleteHolidayRequest={e => this.handleUpdateHolidayRequest(e, true)}
```

Comment:
Just a small thing, but I'm not a fan of reducing variable names to single letters. It's hard to tell what it's referencing when doing a PR review without me diving into the code and checking.